test/libsolidity/syntaxTests/inheritance/override/ambiguous_base_and_unique_mention.sol

```diff
@@ -0,0 +1,17 @@
+interface I {
+    function f() external;
+    function g() external;
+}
+interface J {
+	function f() external;
+}
+abstract contract A is I, J {
+    function f() external override (I, J) {}
+}
+abstract contract B is I {
+    function g() external override {}
+}
+contract C is A, B {
+}
+// ----
+// TypeError: (254-276): Derived contract must override function "f". Function with the same name and parameter types defined in two or more base classes.
```

Comment:
Everyone agree that this needs to be overridden? ``A`` is not a suitable common base, since there is still a path through ``B`` to ``I``. Neither is ``I``, since ``f`` is also in ``J``.

Reply:
We should check the specification against this case, though.

Reply:
Well, but `I` doesn't implement `f`. Wasn't the spec only considering paths to implementations?

Reply:
All override paths - whether the function is implemented or not doesn't matter.